src/js/models/tool.js

```diff
@@ -153,5 +153,11 @@ define(function (require, exports, module) {
      */
     Tool.prototype.toolOverlay = null;
 
+    /**
+     * Optional indicate is a tool is usable in vector make mode 
+     * @type {boolean}
+     */
+    Tool.prototype.handlevectorMaskMode = null;
```

Comment:
I disagree that it shouldn't be declared here as well, can you elaborate on why you think that is a better choice.